projects/vue/jest.config.js

```diff
@@ -6,4 +6,7 @@ module.exports = {
     },
     moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
     coverageDirectory: '../../coverage/projects/vue',
+    testEnvironmentOptions: {
+        customExportConditions: ['node', 'node-addons'],
```

Comment:
https://jest-archive-august-2023.netlify.app/docs/28.x/upgrading-to-jest28/#packagejson-exports